ble_controller/include/ble_controller_hci_vs.h

```diff
@@ -443,6 +452,27 @@ uint8_t hci_vs_cmd_conn_event_extend(const hci_vs_cmd_conn_event_extend_t * p_pa
  */
 uint8_t hci_vs_cmd_qos_conn_event_report_enable(const hci_vs_cmd_qos_conn_event_report_enable_t * p_params);
 
+/** @brief Set event length for connections.
+ *
+ * That is, if this API must be called before starting a connectable advertiser or connecting to an
```

Comment:
"This API can be called before starting a connectable advertising or connecting to an advertiser"

Reply:
I agree that this sentence needs work, starting with "That is, " is not good.
Also link should not be mixed in here:

How about:
"Set the event length for new connections. This API must be called before starting a connectable advertiser or starting an initiator for the event length to applied to the connection once established."